pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -3,7 +3,7 @@
 , which, python, perl, gnused, gnugrep, findutils
 # Always assume all markers valid (don't redownload dependencies).
 # Also, don't clean up environment variables.
-, enableNixHacks ? false
+, enableNixHacks ? false, build_jdk ? jdk, run_jdk ? jdk
```

Comment:
The jdk stuff should go to a newline to not be confused the comment above.
I think camel case is more commonly used for parameters that are meant to be overridden:
`buildJdk` and `runJdk` - then the it is less likely to collide with existing package names.

Reply:
Sounds good. Reordered the lines and added a comment to make it clearer which are the straightforward dependencies being passed in and which are parameters to change the output of the derivation.